wirecard/oxid-ee
# WDPD-56 Terms of use

### This PR

* Adds the Wirecard terms of use to the module activation page

### How to Test

* log in to the admin area
* click on "Extensions" and "Modules", then select the "Wirecard Oxid EE Paymentgateway"
* in the module description, there should be the text "General Information regarding Wirecard Shop Plugins", with a "(show)" link next to it
* when "(show)" is clicked, the "Terms of use" content should be displayed underneath, and "(show") should switch to "(hide)"
* when "(hide)" is clicked, the content should be removed and "(hide)" should switch back to "(show)"

### Jira Links

* https://jira.parkside.at/browse/WDPD-56
